### ISSUE TITLE
On the Demo: Different image sizes LQIP and Final Image

### ISSUE THREAD

**Issue by willgriffiths** [zouhir/lqip-loader#4]
Refreshing the demo app I can see that for the mountain image the:
The LQIP is 900x386 px
And the final image is 900:400 px

Shouldn't they be the same size? Or is this on purpose? :)

**Comment by zouhir** [comment]
That's a good catch @willgriffiths 

the LQIP is just 14px of width (1.55% of original width) and css is the reason it is stretched to fill 900px. I'm afraid my CSS to stretch the width only is amplifying or showing issues in the ratio.

In a perfect world scenario, ratio should stay the same. But I wouldn't recommend letting the LQIP and image set a dynamic container height _for now only_ until I do more investigations, and I recommend this more real life example https://shop.polymer-project.org/ where they use a pre specified height for the banner div and get the LQIO to fill it 100% in both width and height :)

All other examples I saw in the web use a specific height, but I am willing to investigate the ratio issue and try to get it accurate come back to you in this thread. will keep it open.

Thanks for the catch and hope I helped!
